Gopkg.toml

```diff
@@ -8,15 +8,15 @@
 
 [[constraint]]
   name = "github.com/ethereum/go-ethereum"
-  version = "1.8.21"
+  version = "1.8.23"
```

Comment:
don't we have to coordinate such a change with Leth as well? Did you check if the 2 versions can play together plz?

Reply:
That is already part of my TODO list.  But I was waiting for the PR to be approved before I update `leth`

Reply:
I will update `leth` once this changes are approved. It was already on my TODO list

Reply:
great